Documentation/learning/api.md

```diff
@@ -47,7 +47,7 @@ message ResponseHeader {
 
 An application may read the Cluster_ID (Member_ID) field to ensure it is communicating with the intended cluster (member).
 
-Applications can use the `Revision` to know the latest revision of the key-value store. This is especially useful when applications specify a historical revision to make time `travel query` and wish to know the latest revision at the time of the request.
+Applications can use the `Revision` to know the latest revision of the key-value store. This is especially useful when applications specify a historical revision to make a `time travel query` and wish to know the latest revision at the time of the request.
```

Comment:
add "field" after `Revision` for consistency and better reading i.e. Applications can use the `Revision` field to...